docs/notebooks/Calculating ground states on large scale systems with the quantum krylov subspace method.ipynb

```diff
@@ -0,0 +1,1037 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Calculating ground states on large scale systems: Quantum Krylov Subspaces"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 1: Map problem to quantum native format\n",
+    "\n",
+    "Given a matrix H that we are interested in, subspace methods use a set of states as a basis for the construction of a smaller representation of H, which captures its properties of interest (e.g. lowest eigenvalue).\n",
+    "\n",
+    "\n",
+    "What is the Krylov subspace? \n",
+    "\n",
+    "$$K^r = \\left\\{ \\vert \\psi \\rangle, H \\vert \\psi \\rangle, H^2 \\vert \\psi \\rangle, ..., H^{r-1} \\vert \\psi \\rangle \\right\\}$$\n",
+    "\n",
+    "Is the Krylov subspace for a given matrix $H$ and vector $\\vert \\psi \\rangle$ of order $r$ where $\\vert \\psi \\rangle$ is an aribitrary state called \"reference state\".\n",
+    "\n",
+    "The reference state can be expanded in terms of the eigenvectors $\\vert \\lambda_i \\rangle$ of the matrix $H$:\n",
+    "\n",
+    "$$ \\vert \\psi \\rangle = c_1 \\vert \\lambda_1 \\rangle + c_2 \\vert \\lambda_2 \\rangle + ... + c_n \\vert \\lambda_n \\rangle $$\n",
+    "\n",
+    "Applying $j^{th}$ power of the matrix $H$ gives:\n",
+    "\n",
+    "$$ H^n \\vert \\psi \\rangle = c_1 \\lambda_1^n \\vert \\lambda_1 \\rangle + c_2 \\lambda_2^n \\vert \\lambda_2 \\rangle + ... + c_n \\lambda_n^n \\vert \\lambda_n \\rangle $$\n",
+    "\n",
+    "Which means that the component $k$ with the largest eigenvalue $\\lambda_k$ is amplified by the power iteration (This can also be a problem as the basis vector become too similar to each other). The same is true for the smallest eigenvalue, if we consider power iteration of the matrix $H^{-1}$.\n",
+    "\n",
+    "Why is it useful for ground state energy problems?\n",
+    "\n",
+    "The Krylov subspace is constructed using the power iteration method. Therefore, states in the Krylov subspace corresponding to the multiplication with higher power of the matrix with the reference states will have the contribution of the ground state $\\vert \\lambda_k \\rangle$ enhanced.\n",
+    "\n",
+    "\n",
+    "The Krylov subspace that we use classically cannot be accessed on a quantum computer as $H$ is not a unitary matrix. Instead, we can use the time-evolution operator $U = e^{-iHt}$ which can be shown to give similar convergence guarantess as the power method. Powers of $U$ then become different time steps $U^k = e^{-iH(kt)}$.\n",
```

Comment:
add a reference to your statement "... which can be shown to give similar convergence guarantess as the power method."

Also a typo in **guarantess**

Reply:
Thanks, this should be addressed now